eng/pipelines/templates/steps/analyze.yml

```
@@ -8,6 +8,16 @@ parameters:
 
 steps:
 
+  - task: Powershell@2
+    displayName: 'Dependency Check'
+    env:
+      GO111MODULE: 'on'
+    inputs:
+      targetType: filePath
+      pwsh: true
+      filePath: eng/scripts/Invoke-DependencyCheck.ps1
+      arguments: '${{ parameters.ServiceDirectory }}'
```

For consistency should we pass in the `sdk/` part of this as well?
```suggestion
      arguments: 'sdk/${{ parameters.ServiceDirectory }}'
```